server/src/main/java/org/elasticsearch/action/admin/indices/stats/IndicesStatsResponse.java

```diff
@@ -185,13 +186,17 @@ public void writeTo(StreamOutput out) throws IOException {
     }
 
     @Override
-    protected Iterator<ToXContent> customXContentChunks(ToXContent.Params params) {
+    protected Iterator<ToXContent> customXContentChunks(ToXContent.Params outerParams) {
+        if (outerParams.param(DenseVectorStats.INCLUDE_OFF_HEAP) == null) {
+            outerParams = new ToXContent.DelegatingMapParams(Map.of(DenseVectorStats.INCLUDE_OFF_HEAP, "true"), outerParams);
+        }
```

Comment:
For my own sanity, does this mean we default to returning off-heap information?

Reply:
yes.